staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -730,7 +726,7 @@ func (s *store) GetList(ctx context.Context, key string, opts storage.ListOption
 
 		// take items from the response until the bucket is full, filtering as we go
 		for i, kv := range getResp.Kvs {
-			if paging && int64(v.Len()) >= opts.Predicate.Limit {
```

Comment:
this bool still looks relevant since this block only triggered when `opts.Predicate.Limit > 0` previously

Reply:
done